src/main/java/org/immregistries/mqe/hub/submission/IISGateway.java

```diff
@@ -0,0 +1,122 @@
+package org.immregistries.mqe.hub.submission;
+
+import java.util.Arrays;
+import java.util.List;
+import org.immregistries.mqe.hl7util.Reportable;
+import org.immregistries.mqe.hl7util.ReportableSource;
+import org.immregistries.mqe.hl7util.SeverityLevel;
+import org.immregistries.mqe.hl7util.builder.AckData;
+import org.immregistries.mqe.hl7util.builder.HL7Util;
+import org.immregistries.mqe.hl7util.model.CodedWithExceptions;
+import org.immregistries.mqe.hl7util.model.Hl7Location;
+import org.immregistries.mqe.hub.rest.model.Hl7MessageSubmission;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class IISGateway {
+
+  private static final boolean ENABLED = false;
+
+  //  public static final String IIS_GATEWAY_URL = "http://florence.immregistries.org/iis-kernel/pop";
+  public static final String IIS_GATEWAY_URL = "http://localhost/iis-kernel/pop";
```

Comment:
I think this is really cool, and one way to make it more configurable is to use an App property that it can read from the  application.properties file. One way to do that is to annotate it with the @value option like this:  
```suggestion
  @Value("${iisgateway.url}")
  private String IIS_GATEWAY_URL = "http://localhost/iis-kernel/pop";
```
I think that will allow it to be pulled from application properties under an entry like this: 
iisgateway.url="http://localhost/iis-kernel/pop"